addons/godot-xr-tools/functions/function_pointer.gd

```diff
@@ -40,7 +40,7 @@ var ws = 1.0
 
```

Comment:
Shouldn't function_pointer get a class name?

Reply:
Modified to "class_name XRToolsFunctionPointer" with icon.